requirements.txt

```diff
@@ -1,7 +1,9 @@
-jsonschema>=2.5.1
-flask>=0.10.1
-PyYAML>=3.11
-requests>=2.9.1
-six>=1.7
-strict-rfc3339>=0.6
-swagger_spec_validator>=2.0.2
+jsonschema>=2.5.1,<3
+flask>=0.10.1,<0.12
+PyYAML>=3.11,<4
+requests>=2.9.1,<3
```

Comment:
`requests` [uses semantic versioning](http://docs.python-requests.org/en/master/dev/philosophy), so if the major version number does not change then that's an explicit promise that the code is back-compatible.